koku/masu/processor/oci/oci_cost_model_cost_updater.py

```diff
@@ -47,7 +48,7 @@ def _update_markup_cost(self, start_date, end_date):
                     bill_ids = [str(bill.id) for bill in bills]
                 report_accessor.populate_markup_cost(markup_value, start_date, end_date, bill_ids)
         except OCICostModelCostUpdaterError as error:
-            LOG.error("Unable to update markup costs. Error: %s", str(error))
+            LOG.error(log_json(msg="unable to update markup costs."), exc_info=error)
```

Comment:
```suggestion
            LOG.error(log_json(msg="unable to update markup costs"), exc_info=error)
```
The nitpickiest of all nitpicks 😂